sourced/ml/__main__.py

```diff
@@ -66,6 +66,22 @@ def add_default_args(my_parser):
         choices=[ex.NAME for ex in extractors.__extractors__.values()],
         required=True, help="The feature extraction scheme to apply.")
 
+    repos2ids_parser = subparsers.add_parser(
+        "repos2ids", help="Convert source code to a token identifier stream."
+                           "The identifiers are splittable on special characters"
+                           "or case changes")
+    repos2ids_parser.set_defaults(handler=repos2ids_entry)
+    add_engine_args(repos2ids_parser)
+    repos2ids_parser.add_argument(
+        "-r", "--repositories", required=True,
+        help="The path to the repositories.")
+    repos2ids_parser.add_argument(
+        "--token-stream", required=True,
+        help="[OUT] The path to the token stream of identifiers.")
+    repos2ids_parser.add_argument(
+        "--vocabulary-size", default=10000000, type=int,
+        help="The maximum vocabulary size.")
```

Comment:
You also need to filter only those which are more than 1 part. We should add a flag.

Reply:
@vmarkovtsev , I don't understand the purpose of such a flag. I thought there was no other option: we are always going to filter tokens with more than 1 part.

Reply:
The name of your command is `repos2ids`. You generate unique identifiers from source code. You do not dump them splitted or X-Y-ed. Filtering 1-part tokens is a part of your specific application to train the model. We can perform that action later and can perform it here to save some time and space. However, we do not have to.

Reply:
Yes I get it now, thx!